epsagon/events/greengrasssdk.py

```diff
@@ -0,0 +1,77 @@
+"""
```

Comment:
looks good : ) 
can you add a sanity UT test?

Reply:
👍